### ISSUE TITLE
Cannot restore packages from teamcity nuget server

### ISSUE THREAD

**Issue by dtretyakov** [JetBrains/teamcity-nuget-support#19]
_From @prabhakarreddy1234 on September 28, 2016 10:29_

I am using teamcity as my nuget feed. I published 1  dotnet core package using TeamCity CI. I can see that  nuget package has been dropped into the nuget feed folder in Teamcity. But cannot restore that package using visual studio or command line. 

I used `dotnet pack` command.

![image](https://cloud.githubusercontent.com/assets/1984707/18910182/c091ffb8-856e-11e6-8bda-53e90a254314.png)

_Copied from original issue: JetBrains/teamcity-dotnet-plugin#19_

**Comment by dtretyakov** [comment]
@prabhakarreddy1234, which TeamCity version do you use to host nuget feed?
Could you please navigate to the following link and ensure that output contains your package:
```
%teamcity%/guestAuth/app/nuget/v1/FeedService.svc/FindPackagesById()?Id=%27API.Framework.Entities.Core%27
```

If response contains your package, please define [internal property](https://confluence.jetbrains.com/display/TCDL/Configuring+TeamCity+Server+Startup+Properties#ConfiguringTeamCityServerStartupProperties-TeamCityinternalproperties) `teamcity.nuget.feed.filter.targetframework=true` and try to restore packages again.

**Comment by prabhakarreddy1234** [comment]
I am using `TeamCity Enterprise 9.1.6 (build 37459)` .

I tried to access the link which you provided but getting the below response. 

![image](https://cloud.githubusercontent.com/assets/1984707/18911036/a5ed7a94-8572-11e6-8f3d-8f870348b80a.png)

**Comment by dtretyakov** [comment]
@prabhakarreddy1234, ok, it seems that you have disabled guest authentication in your feed.
In this case navigate to the `%teamcity%/httpAuth/app/nuget/v1/FeedService.svc/FindPackagesById()?Id=%27API.Framework.Entities.Core%27`

**Comment by prabhakarreddy1234** [comment]
Actually earlier i tried with `httpAuth`.  Now i tried again with `guestAuth`. Still i cannot see the package in the response.  I am getting  HTTP 204  ( No content).    I have other package which is `API.Framework.Entities`  it is on Full .Net Framework.   I can get response for this package.

**Comment by dtretyakov** [comment]
@prabhakarreddy1234, thanks for details. It seems like a bug in TeamCity 9.1.x.

Latest [TeamCity 10.0.2 version](https://www.jetbrains.com/teamcity/download) contains [lots of nuget-related fixes and improvements](https://confluence.jetbrains.com/display/TW/TeamCity+10.0.2+(build+42234)+Release+Notes). 
So, it would be great to [upgrade](https://confluence.jetbrains.com/display/TCDL/Upgrade) your TeamCity installation.

If could not do that, you can try to [latest nuget plugin version](https://teamcity.jetbrains.com/repository/download/TeamCityPluginsByJetBrains_NuGet_NuGetSupportV014for100x/868941:id/dotNetPackagesSupport.zip?guest=1) and install it as [additional TeamCity plugin](https://confluence.jetbrains.com/display/TCDL/Installing+Additional+Plugins).

**Comment by prabhakarreddy1234** [comment]
I will try that and let you know the result in a few days.

**Comment by prabhakarreddy1234** [comment]
Latest nuget plugin which you suggested didn't work with TeamCity 9.1.x. So i upgraded to Teamcity 10.0.2.  Now i can see .net core package when i browse `%teamcity%/httpAuth/app/nuget/v1/FeedService.svc/FindPackagesById()?Id=%27API.Framework.Entities.Core%27` .  

But in Visual studio 2015 update 3 , Nuget 3.5.0  i cannot see the package in `Manage Nuget Packages` window when i select `TeamCity` feed. I can see the package if i select my `local folder feed`.    Is there anything you can help me with?

**Comment by dtretyakov** [comment]
@prabhakarreddy1234, could please try to use nuget plugin with a compatibility fix for nuget 3.4+:
https://youtrack.jetbrains.com/issue/TW-47189#comment=27-1645705
